gpkit/nomials.py

```diff
@@ -569,4 +601,80 @@ def __bool__(self):
         return self.__nonzero__()
 
 
+class SignomialConstraint(Signomial):
+    """A constraint of the general form posynomial >= posynomial
+    Stored internally (exps, cs) as a single Signomial (0 >= self)
+    Usually initialized via operator overloading, e.g. cc = (y**2 >= 1 + x - y)
+    Additionally retains input format (lhs vs rhs) in self.left and self.right
+    Form is self.left >= self.right.
+    """
+    def __str__(self):
+        return str(self.left) + self.oper_s + str(self.right)
```

Comment:
we've agreed to make pretty constraint printing a fragile property, so after substituting a SignomialConstraint, it will become either a posynomial (implicitly <= 1) or a signomial (implicitly <= 0). Need to implement this.